product_docs/docs/epas/15/database_administration/02_edb_loader/edb_loader_overview_and_restrictions.mdx

```diff
@@ -21,9 +21,9 @@ EDB\*Loader features include:
 
 The important version compatibility restrictions between the EDB\*Loader client and the database server are:
 
--   When you invoke the EDB\*Loader program (called `edbldr`), you pass in parameters and directive information to the database server. We strongly recommend that you use the version 14 EDB\*Loader client (the edbldr program supplied with EDB Postgres Advanced Server 14) to load data only into version 14 of the database server. In general, use the same version for the EDB\*Loader client and database server.
+-   When you invoke the EDB\*Loader program (called `edbldr`), you pass in parameters and directive information to the database server. We strongly recommend that you use the version 14 EDB\*Loader client (the `edbldr` program supplied with EDB Postgres Advanced Server 14) to load data only into version 14 of the database server. In general, use the same version for the EDB\*Loader client and database server.
```

Comment:
```suggestion
-   When you invoke the EDB\*Loader program (called `edbldr`), you pass in parameters and directive information to the database server. We strongly recommend that you use the version of EDB\*Loader client, the `edbldr` program, supplied with the version of EDB Postgres Advanced Server you are using to load data only into the database server. In general, use the same version for the EDB\*Loader client and database server.
```

Reply:
@nidhibhammar is there anything special about v14? I doubt it. I think this is just legacy content that got rolled over. I think we can make this version agnostic.

Reply:
@tusharahuja, is there anything specific about the edb loader v14 client?

Reply:
Right, there is nothing particular about edb*loader v14. Every EPAS server version comes with edb*loader which is available under the bin folder. Thanks.